packages/manager/modules/netapp/src/dashboard/routing.js

```diff
@@ -38,6 +38,8 @@ export default /* @ngInject */ ($stateProvider) => {
             commitmentAvailability.isFeatureAvailable('billing:commitment'),
           )
           .catch(() => false),
+      schema: /* @ngInject */ ($http) =>
+        $http.get('/storage.json').then(({ data }) => data),
```

Comment:
Yes it is doesn't used in the dashboard for the moment I agree, but what if another page needs a specific enumeration of this schema ? I let it there for such cases.

What do you think ?

Reply:
It is being used in packages/manager/modules/netapp/src/dashboard/volumes/create/routing.js

Reply:
Yes @ganeshkumar1989 and my point is that, instead of limiting it to this state, I've set the resolve in a more general state to allow it to be accessible elsewhere

Reply:
@JeremyDec until it is needed elsewhere it would be better to call the route only when needed because here a user that have no intention of creating a volume will load the schema for no purpose

Reply:
done